File: springdoc-openapi-security/src/test/java/test/org/springdoc/api/app2/SpringDocApp2Test.java
package test.org.springdoc.api.app2;

import io.swagger.v3.oas.models.OpenAPI;
import io.swagger.v3.oas.models.info.Info;
import io.swagger.v3.oas.models.info.License;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import test.org.springdoc.api.AbstractSpringDocTest;

public class SpringDocApp2Test extends AbstractSpringDocTest {

    @SpringBootApplication(scanBasePackages = {"test.org.springdoc.api.configuration,test.org.springdoc.api.app2"})
    static class SpringDocTestApp {
        @Bean
        public OpenAPI customOpenAPI() {
            return new OpenAPI()
                    .info(new Info().title("Security API").version("v1")
                            .license(new License().name("Apache 2.0").url("http://springdoc.org")));
        }
    }

}

File: springdoc-openapi-kotlin/src/test/kotlin/test/org/springdoc/api/app2/SpringDocApp2Test.kt
package test.org.springdoc.api.app2

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan
import test.org.springdoc.api.AbstractKotlinSpringDocTest

@ComponentScan(basePackages = ["org.springdoc", "test.org.springdoc.api.app2"])
class SpringDocApp2Test : AbstractKotlinSpringDocTest() {

    @SpringBootApplication
    open class DemoApplication

}